source/extensions/tracers/xray/tracer.cc

```diff
@@ -38,7 +40,7 @@ std::string generateTraceId(SystemTime point_in_time, Random::RandomGenerator& r
   const auto epoch = time_point_cast<seconds>(point_in_time).time_since_epoch().count();
   std::string out;
   out.reserve(35);
-  out += XRaySerializationVersion;
+  out += std::string(XRaySerializationVersion);
```

Comment:
consider
```
const std::string out = absl::StrCat(XRaySerializationVersion, "-", Hex::uint32ToHex(epoch));
```

Reply:
Refactored this function to utilize `absl::StrCat`. Thanks.